src/generator/AutoRest.Ruby.Azure/Templates/AzureMethodTemplate.cshtml

```diff
@@ -67,6 +67,11 @@ end
 @WrapComment("# ", string.Format("@return [{0}] operation results.", Model.OperationReturnTypeString))@:
 @:#
 }
+else
+{
+@WrapComment("# ", string.Format("@return {0} operation results.", Model.Name))@:

```

Comment:
@sarangan12 could you provide an example of what's being returned? Model.Name seems to be the same as the operation name here, does this match the return type?